src/open_inwoner/components/templates/components/Typography/Link.html

```diff
@@ -17,6 +17,7 @@
     {% endif %}
     {% if toggle_exclusive %}data-toggle-exclusive="{{ toggle_exclusive }}"{% endif %}
     aria-describedby="{{ text }}"
+    {% if hide_text %}aria-hidden="true"{% endif %}
```

Comment:
This means that this link would not be available for screen readers if we want to only hide the text for visual browsers. I think the aria-label should always be populated and the context may be hidden (as long as aria-label is set) but aria-hidden does not represent the visual state of the element:

 > Use caution when using aria-hidden to hide visibly rendered content from assistive technologies. You should not be hiding visible content unless doing so improves the experience for users of assistive technologies by removing redundant or extraneous content. Only when identical or equivalent meaning and functionality is exposed to assistive technologies can removing visible content from the accessibility API be considered.

https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles